### ISSUE TITLE
No request_timeout found in ArangoClient 7.4.1

### ISSUE THREAD

**Issue by DespuesLeeWallace** [ArangoDB-Community/python-arango#219]
Hello, I am trying to configure the `request_timeout` prameter of `ArangoClient` but it is not available in the 7.4.1 version installed through pip.

The constructor looks like this:
```
 def __init__(
        self,
        hosts: Union[str, Sequence[str]] = "http://127.0.0.1:8529",
        host_resolver: str = "roundrobin",
        resolver_max_tries: Optional[int] = None,
        http_client: Optional[HTTPClient] = None,
        serializer: Callable[..., str] = lambda x: dumps(x),
        deserializer: Callable[[str], Any] = lambda x: loads(x),
        verify_override: Union[bool, str, None] = None,
    ) -> None:
```

**Comment by joowani** [comment]
I just released version 7.5.0. Please run `pip install python-arango --upgrade` and try again. Thanks.

**Comment by DespuesLeeWallace** [comment]
Thanks !!!! quite fast
